internal/inventory/accessor_test.go

```diff
@@ -0,0 +1,130 @@
+package inventory
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"log"
+	"testing"
+	"time"
+
+	"kg/procurement/internal/vendors"
+	"github.com/DATA-DOG/go-sqlmock"
+	"github.com/onsi/gomega"
+)
+
+var mockVendor = vendors.Vendor{
```

Comment:
i think it's better to put this on scopes that do actually need it rather than being a global var